eduiddashboard/views/security.py

```diff
@@ -39,14 +41,25 @@
 
 
 def change_password(request, user, old_password, new_password):
-    """ Change the user password, deleting old credentials """
+    """
+    Change the user password, deleting old credentials
+
+    @param request: Request object
+    @param user: User object
+    @param old_password: Old password, if supplied (i.e. not a password reset)
+    @param new_password: New password
+
+    @type request: Request
+    @type user: User
+    @type old_password: str or unicode or None
+    """
     vccs_url = request.registry.settings.get('vccs_url')
+    log.debug("Changing password user {!s}\nTIMESTAMP 1 {!s}".format(user, user.modified_ts))
     added = add_credentials(vccs_url, old_password, new_password, user)
+    log.debug("Changing password user {!s}\nTIMESTAMP 2 {!s}".format(user, user.modified_ts))
```

Comment:
I think (hope) these two log.debug are just leftovers from debugging the retrieve_modified_ts changes. Remove, or keep for a little while in case this branch introduces issues with the syncing?